admin_actions/admin.py

```diff
@@ -297,6 +297,8 @@ def _confirmation_view(
         form = None
         opts = self.model._meta
         obj = self.get_object(request, pk)
+        if not obj:
+            raise Http404()
```

Comment:
```suggestion
        if pk is not None and obj is None:
            raise Http404()
```
in future we can adopt this method for confirmation of list view actions